python/lsst/daf/base/baseLib.i

```diff
@@ -161,6 +161,48 @@ def _propertyContainerGet(container, name, asArray=False):
         pass
     raise lsst.pex.exceptions.TypeError('Unknown PropertySet value type for ' + name)
 
+import numbers
+def _guessIntegerType(container, name, value):
+    """Given an existing container and name, determine the type
+    that should be used for the supplied value. The supplied value
+    is assumed to be a scalar.
+
+    On Python 3 all ints are LongLong but we need to be able to store them
+    in Int containers if that is what is being used (testing for truncation).
+    Int is assumed to mean 32bit integer (2147483647 to -2147483648).
```

Comment:
It may well be on most systems these days, but are we going to get bitten by assuming an architecture?  Does `PropertySet` define it as a particular size, or just `int` ("at least 16 bits").